RecoLocalTracker/SiPixelRecHits/interface/PixelCPEFast.h

```diff
@@ -31,6 +31,8 @@ class PixelCPEFast final : public PixelCPEGenericBase {
 
   pixelCPEforGPU::ParamsOnGPU const &getCPUProduct() const { return cpuData_; }
 
+  bool const isUpgrade() const { return isUpgrade_; };
```

Comment:
`const` on return value here does not enforce anything and is mostly confusing
```suggestion
  bool isUpgrade() const { return isUpgrade_; };
```

Reply:
Indeed shouldn't be there: missed it in the last commit, will remove it in the next commit for the following reviews.